oauthdoctor/diag/config.go

```diff
@@ -188,10 +188,8 @@ func (c *ConfigFile) ReplaceConfigFromReader(key, value string, r io.Reader) str
 		langKey := c.GetConfigKeysInLang(key)
 
 		// Found the line with old config key and comment it out
-		if c.Lang == "dotnet" {
-			if strings.Contains(trimmedLine, langKey) {
-				buf.WriteString("<!-- " + trimmedLine + " -->\n")
-			}
+		if c.Lang == "dotnet" && strings.Contains(trimmedLine, langKey) {
```

Comment:
Why do you not need
!strings.HasPrefix(trimmedLine, commentChar)
for the case of dotnet?

If the lang is dotnet, and it contains the language key, we don't need to first test whether the string contains the comment characters?

Reply:
This line is already in master branch. GitHub flagged it for no reason. The overall fix should be done in #21. This branch is mainly for unit testing